hass_nabucasa/__init__.py

```diff
@@ -37,6 +35,10 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+if TYPE_CHECKING:
```

Comment:
I don't like this rule, and I'd recommend we disable it. It's just annoying to have to put some imports here when there's no real reason. I don't believe the performance benefit is worth it.

Reply:
OK